docs/develop-on-a-remote-host.md

```diff
@@ -161,19 +161,12 @@ This section describes how to instantiate an AWS EC2 as the remote host.
 
 - Update system packages
    ```console
-   sudo apt update && sudo apt upgrade -y
-   ```
-- [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository)
-- Add your user account to docker group.
-   ```console
-   sudo usermod -aG docker ${USER}
```

Comment:
`sudo usermod -aG docker ${USER}` I ran it during the workshop, so I am not sure if this is still needed. @vpchung 
 Did you need to run this CLI?

Reply:
I never had to run this when I was setting up, since the instance image already configured Docker appropriately, so I'm not sure if this is needed?

I thought the error you encountered earlier was from the fork being out of sync?